bin/push-docker

```diff
@@ -24,13 +24,24 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-hub="gcr.io/istio-testing"
-tags=$(whoami)_$(date +%y%m%d_%h%m%s)
+
+function docker_push() {
+  im="${1}"
+  if [[ "${im}" =~ 'gcr.io' ]]; then
+    gcloud docker -- push "${im}"
```

Comment:
I removed this. I think those accomplish the same thing. In any case you have to call gcloud for gcr.io images, so what s the issue ?